src/helics/helics_enums.h

```diff
@@ -305,6 +305,13 @@ typedef enum {
 
 } helics_filter_type;
 
+/** enumeration of sequencing modes for queries
+fast is the default meaning the query travels along priority channels and takes precedence of over
+existing messages ordered means it follows normal priority patterns and will be ordered along with
+existing messages
+*/
+typedef enum { helics_query_mode_fast = 0, helics_query_mode_ordered = 1 } query_synch_mode;
```

Comment:
```suggestion
typedef enum { helics_query_mode_fast = 0, helics_query_mode_ordered = 1 } query_sync_mode;
```

Thoughts on renaming this to `sync` instead of `synch`? I think `sync` is the more common abbreviation.

Reply:
I am renaming it to helics_query_mode,  I was originally running async vs synchronous but as I got further down that made a little less sense,  I am also adding it to the C++98 and C interfaces.